Formula/semgrep.rb

```diff
@@ -71,7 +75,7 @@ def install
         system "opam", "exec", "--", "make", "all"
         system "opam", "exec", "--", "make", "install"
         build_dir = "_build/default/bin/"
-        File.rename(build_dir + "main_semgrep_core.exe", build_dir + "semgrep-core")
+        File.rename(build_dir + "Main.exe", build_dir + "semgrep-core")
         bin.install build_dir + "semgrep-core"
```

Comment:
It looks like we could simplify it something like this:

```suggestion
        bin.install "_build/default/bin/Main.exe" => "semgrep-core"
```

Reply:
And could you please squash all the commits into one `semgrep 0.8.1`
Thanks!